src/components/Modals/ExpertOverviewModal.tsx

```diff
@@ -86,12 +86,27 @@ export const ExpertOverviewModal: React.FC = () => {
     >
       <div className={classes.modal}>
         <Title size="h2">{MODAL_TITLE}</Title>
-        <Search
-          placeholder="Suche nach Expert*innen für.."
-          allowClear
-          onSearch={onSearch}
-          className={classes.search}
-        />
+        {experts && (
+          <Select
+            showSearch
+            style={{ width: '100%', marginBottom: '1rem' }}
+            placeholder="Suche nach Expert*innen für.."
+            optionFilterProp="children"
+            onChange={onSearch}
+            filterOption={(input, option) =>
+              option.children
+                .toString()
+                .toLowerCase()
+                .indexOf(input.toLowerCase()) >= 0
+            }
+          >
+            {experts.map((val) => (
+              <Option value={`${val.firstName}`} key={`key-${val.firstName}`}>
```

Comment:
From the user perspective it would make more sense to get proposals for tags or project fields etc. Also refer to `onSearch` to see what is actually searched. The expert's names seem least interesting at this place.